### ISSUE TITLE
Debian 11: Network Mode Emulated VLAN prevents connections to Internet

### ISSUE THREAD

**Issue by LukBukkit** [utmapp/UTM#4334]
**Describe the issue**  
Debian 11.4.0 ([arm64](https://cdimage.debian.org/debian-cd/current/arm64/iso-cd/debian-11.4.0-arm64-netinst.iso)) can't connect to the Internet (an external network) when the network mode *Emulated VLAN* is selected. It works fine with the network mode *Shared Network*. [Debian 10](https://mac.getutm.app/gallery/debian-10-4-minimal) from the Gallery utilizes the network mode *Emulated VLAN* without any problems.

**Configuration**  
* UTM Version: 3.2.4
* macOS Version: 12.5.1
* Mac Chip (Intel, M1, ...): M1 Pro

**Debug log**  
[debug.log](https://github.com/utmapp/UTM/files/9410821/debug.log)

**Upload VM**  
I can upload the entire .utm file if required.
[config.plist.zip](https://github.com/utmapp/UTM/files/9410825/config.plist.zip)

**Comment by aeciolevy** [comment]
@LukBukkit any luck making it work? 

I have got the same issue with Ubuntu 22. _Shared network_ internet works just fine, but _Emulated VLAN_ internet does not work. I have switched to _Emulated VLAN_ just to access it through ssh...

**Comment by LukBukkit** [comment]
No, sadly I didn't get any further with the resolution of this issue. I just keep using the *Shared Network* mode.

**Comment by aeciolevy** [comment]
Can you access it thought ssh?

**Comment by LukBukkit** [comment]
I've deleted the VM a few days ago, but it was possible to access it via SSH. You have to find out its IP address. The command `ip addr show` should be helpful for this. Then, you can connect to the VM via SSH from your Mac with `ssh user@ip`.
